src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -238,6 +238,13 @@ public Player get(final BlockPlaceEvent e) {
 				return e.getPlayer();
 			}
 		}, 0);
+		EventValues.registerEventValue(BlockPlaceEvent.class, ItemStack.class, new Getter<ItemStack, BlockPlaceEvent>() {
+			@Override
+			@Nullable
+			public ItemStack get(final BlockPlaceEvent e) {
+				return e.getItemInHand();
```

Comment:
use event instead of e

Reply:
Should I change the rest of the `e`s while I'm here?

Reply:
Yes, anything use the value `e` for the event should be changed to `event`

Reply:
That would be for another pull request. It's a coding standard, ignore old code, it'll be updated eventually like all things do. We're still forever removing the final abuse.

Reply:
Speaking of, you don't need `final` in the method.

Reply:
Ah, I missed that, led astray by the surrounding code lol